bedrock/settings/base.py

```diff
@@ -1326,6 +1336,22 @@ def JINJA_CONFIG():
         ),
         'output_filename': 'js/firefox_os-bundle.js',
     },
+    'firefox_os_new': {
+        'source_filenames': (
+            'js/base/mozilla-modal.js',
+            'js/libs/jquery.waypoints.min.js',
+            'js/libs/jquery.waypoints-sticky.min.js',
+            'js/libs/tweenmax.1.9.7.min.js',
```

Comment:
I don't see any reason why this new design should need to reference code from the old pages? I don't think the get_device include should have any dependencies on these files. Ideally, logic for the phone/signup should now be in `firefox-so-new.js`